lib/Experian/IDAuth.pm

```diff
@@ -352,7 +352,7 @@ sub _get_result_proveid {
         || 0;
     $decision->{num_verifications} = $cr_total;
 
-    if ($kyc_dob or $cr_total) {
+    if ($kyc_dob and $cr_total) {
```

Comment:
The new logic is that we need both - DoB match AND TotalNumberOfVerifications>0.
Previous logic was that any of this is enough